src/main/java/frameworksdrivers/DatabaseDriver.java

```diff
@@ -20,10 +20,6 @@ public DatabaseDriver() {
         studentDbGateway = new StudentDbGateway(dbConnection);
         sessionDbGateway = new SessionDbGateway(dbConnection);
         courseDbGateway = new CourseDbGateway(dbConnection);
-
-        // Deletes previous sessions that might've not deleted if exited the program wrong.
-        sessionDbGateway.deleteStudentSession();
-        sessionDbGateway.deleteCourseSession();
```

Comment:
This is no longer required here since the sessions are deleted upon closing the application window. 

It is actually a much better implementation because the current way we have, every time that you create a DatabaseDriver, you have to delete the session information. However after the first instance creation of this class, calling these methods won't do anything because the session information is already deleted.